netutils/ftpc/ftpc_connect.c

```diff
@@ -276,5 +276,6 @@ int ftpc_reconnect(FAR struct ftpc_session_s *session)
 errout_with_socket:
   ftpc_sockclose(&session->cmd);
 errout:
+  wd_cancel(&session->wdog);
```

Comment:
Maybe better to add this in an error case of socket connect since watchdog is already canceled in case of other errors?

Reply:
Ok. L165 can be fixed by reordering of wd_start and socket init. But anyway let's keep it with is_active way